src/screens/mealDetails/MealDetails.tsx

```diff
@@ -77,40 +100,43 @@ const TabContent = ({activeTab, recipeInfo}: TabContentProps) => {
     </View>
   );
 
-  const recipeDetails = [
-    {label: 'Servings', value: recipeInfo.servings},
-    {
-      label: 'Ready in',
-      value: recipeInfo.readyInMinutes
-        ? `${recipeInfo.readyInMinutes} minutes`
-        : null,
-    },
-    {
-      label: 'Cooking time',
-      value: recipeInfo.cookingMinutes
-        ? `${recipeInfo.cookingMinutes} minutes`
-        : null,
-    },
-    {
-      label: 'Preparation time',
-      value: recipeInfo.preparationMinutes
-        ? `${recipeInfo.preparationMinutes} minutes`
-        : null,
-    },
-    {label: recipeInfo.vegan ? 'Vegan' : 'Non-Vegan', value: ''},
-    {
-      label: recipeInfo.vegetarian ? 'Vegetarian' : 'Non-Vegetarian',
-      value: '',
-    },
-    {
-      label: recipeInfo.dairyFree ? 'Non-Dairy' : 'Contains Dairy products',
-      value: '',
-    },
-    {
-      label: recipeInfo.glutenFree ? 'Gluten free' : 'Contains Gluten',
-      value: '',
-    },
-  ];
+  const recipeDetails = useMemo(
+    () => [
```

Comment:
this is not readable, you can define the separate function outside component for this all and call inside useMemo passing the `recipeInfo`